### PR TITLE
Upating docker-phpqa tutorial

### PR DESCRIPTION
**Pull request by ghost**
* Adding generate command information;
* Normalizing tutorial with official documentation.

### CODE REVIEW

**Comment by ramsey** [comment]
Please add an `updated` property to the YAML front-matter and put today's date. This way, any feed readers that are following <https://phptestfest.org/feed.xml> will properly pick up the changes.

File: docs/_tutorials/phpqa-tutorial.md [hunk 1]
```diff
@@ -12,48 +13,84 @@ image:
     title: Thousand Colors by Lucas Gallone
     credit: Photo by [Lucas Gallone](https://unsplash.com/photos/PVw_vtpCGaM).
 summary: >
-    This is a small tutorial on how to use PHPQA, a set of Docker tools to easily create and run tests for the PHP-SRC. The intent of this library is to help any PHP developer to create PHPT tests for the language engine and its core extensions.
+    Docker tools to easily create and run tests for the [PHP-SRC][php-src]. The intent of this library is to help any PHP
```
Comment (ramsey) [inline docs/_tutorials/phpqa-tutorial.md]: Missing `php-src` link.

File: docs/_tutorials/phpqa-tutorial.md [hunk 2]
```diff
@@ -12,48 +13,84 @@ image:
...
-**Start PHPTesting!!!**
+Start PHPTesting!!!
 
 ## How to use it?
 
-* Write a PHPT test in any folder;
-* Run the `phpqa` command passing as arguments the filepath (relative or full) and the PHP version to run the test (`72`, `71`, `70`, `56`, `55`):
+### To generate new PHPT test files
+
+The [generate-phpt][generate-phpt] tool wrapped by `docker-phpqa` is a developer tool
```
Comment (ramsey) [inline docs/_tutorials/phpqa-tutorial.md:35]: Missing `generate-phpt` link.

File: docs/_tutorials/phpqa-tutorial.md [hunk 3]
```diff
@@ -12,48 +13,84 @@ image:
...
-# With a version the test will run against the specified version
-phpqa phpt/examples/closure_061.phpt 56
+````bash
+phpqa generate -h
+````
 
-# If you want you can pass `all` as second argument and the test will run against all versions
-phpqa phpt/examples/closure_061.phpt all
-```
+If you want to create a *basic* PHPT test for the [`ucfirst` function][php-function-ucfirst]:
```
Comment (ramsey) [inline docs/_tutorials/phpqa-tutorial.md]: Missing `php-function-ucfirst` link.

**Review by ramsey — CHANGES_REQUESTED** [review]
It looks like you forgot a few links. :smile:

**Comment by ramsey** [comment]
This is how those links are rendering. You just need to add them to the Markdown somewhere.

generate-phpt-link

**Comment by ramsey** [comment]
I know it's weird, but since the `php-src` link appears in the `summary` property of the YAML front-matter, Jekyll doesn't appear to know to grab the named `php-src` link from the bottom of the file. See screenshot here:

php-src-link

Please update the `summary` with the link inline (i.e., `[Link](http://example.com)`), and I'll get this merged in. Everything else looks great.

Thanks!

**Comment by ghost** [comment]
Thanks for reviewing. =)

**Review by ramsey — APPROVED** [review]

**Pull request merged**